src/main/webapp/script.js

```diff
@@ -110,6 +110,719 @@ function createCommentElement(comment) {
   return commentElement;
 }
 
+/** Gets the ID of the YouTube video that the user inputs. */
```

Comment:
You might consider splitting the scripts into multiple files and one main file loading all the scripts.

Reply:
Will split in one PR after everyone's features are in (i.e. end of week 11) to avoid mess. Made a note on the timeline for doing this!